### DISCUSSION TITLE
After I set up the deploy my Hubs to AWS, am I also going to need to connect my route 53 to my s3 bucket or does the process do the same thing?

### DISCUSSION THREAD

**Discussion by ybinstock** [mozilla/hubs#3259] [Hubs Cloud Questions] [answered]
If not, after I set this up and go to my URL, I should see everything working - right?

If I do need to set up an S3 Bucket as well, is there anything, in particular, I have to do or do I just follow the normal set up?

**Comment by robinkwilson** [comment]
No additional setup required other than filling in the Stack Parameters. AWS Cloudformation does all the setup automatically. Good question!

**Reply by ybinstock** [reply]
Hey Robin, then I'm a little confused. 

Where do I post / upload my custom code?

In all three videos, I didn't see a place to upload code where I've customized the UI and the routing and some other functionality. Maybe I missed it...

As a side note, I've tried uploading my code to an s3 bucket and it says it's going to take over a day to upload. 

Is there a reason why that should be taking so long? 

<img width="704" alt="Screen Shot 2020-11-16 at 4 16 50 PM" src="https://user-images.githubusercontent.com/8645066/99271827-5af87500-2827-11eb-8c00-5485f8136686.png">


Please let me know where I can upload my custom Hubs code - thanks!

PS. Thanks for making part 3 of the youtube series for aws.

**Reply by robinkwilson** [reply]
You do not need to manually upload the code! We have a script that deploys the code for you!
https://hubs.mozilla.com/docs/hubs-cloud-custom-clients.html

The `npm run login`, then `npm run deploy` is the command that sends + deploys the code to your hub.

**Comment by robinkwilson** [comment, marked as answer]
You do not need to manually upload the code! We have a script that deploys the code for you!
https://hubs.mozilla.com/docs/hubs-cloud-custom-clients.html

The `npm run login`, then `npm run deploy` is the command that sends + deploys the code to your hub.
